### DISCUSSION TITLE
Different branch prefixes for different managers?

### DISCUSSION THREAD

**Discussion by maeghan-porter** [renovatebot/renovate#32084] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

github actions

### Please tell us more about your question or problem

I have a repo that builds EKS clusters and on creation/push to a branch it will build an ephemeral EKS cluster unless that branch is prefixed with `no-env/`. That said, we are using Renovate on this repo to both keep our Flux helm releases updated as well as keep our github actions up to date. We do not want to build ephemeral clusters for the github actions update but I'm struggling to figure out how to accomplish this and would really love some advice! I have tried using `packageRules` but that appears to be creating `renovate/` branches which we don't want. This is my current config file:
```json
{
    "username": "renovate",
    "gitAuthor": "Renovate Bot <bot@renovateapp.com>",
    "dryRun": "false",
    "forkProcessing": false,
    "onboarding": false,
    "platform": "github",
    "packageRules": [
        {
            "groupName": "flux",
            "matchFileNames": ["flux/**"],
            "enabledManagers": ["flux"],
            "branchPrefix": "rv-",
            "hashedBranchLength": 9
        },
        {
            "groupName": "github-actions",
            "matchFileNames": [".github/workflows/**"],
            "enabledManagers": ["github-actions"],
            "branchPrefix": "no-env/rv-"
        }
    ],
    "branchConcurrentLimit": 5,
    "prConcurrentLimit": 5,
    "pruneStaleBranches": false
}
```

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Potentially you can solve this using both branchPrefix and additionalBranchPrefix together

**Reply by maeghan-porter** [reply]
I looked at `additionalBranchPrefix` but unfortunately it appends at the wrong spot, after `branchPrefix` when it needs to be before it.

**Comment by rarkins** [comment, marked as answer]
Renovate has a restriction of a single `branchPrefix` for the whole repo. This is probably possible to loosen, but it would be quite a bit of work to make sure it doesn't have any undesirable side effects, so it's not currently planned/prioritized.

**Reply by maeghan-porter** [reply]
Well that's unfortunate. Ok thanks, I guess I'll figure something else out.
